master/buildbot/test/unit/test_db_logs.py

```diff
@@ -143,7 +143,7 @@ def compressLog(self, logid, force=False):
 
     def test_signature_deleteOldLogChunks(self):
         @self.assertArgSpecMatches(self.db.logs.deleteOldLogChunks)
-        def deleteOldLogChunks(self, older_than_timestamp):
+        def deleteOldLogChunks(self, older_than_timestamp, exceptions=[]):
```

Comment:
Shouldn't we use empty map here?

Reply:
There's no need to since I just filtering by name. The function doesn't need to know the specific logs; that's handled in the other function.

Reply:
Okay, I got confused.